core/domain/event_services.py

```diff
@@ -76,12 +76,23 @@ def _notify_continuous_computation_listeners_async(cls, *args, **kwargs):
 
     @classmethod
     def _handle_event(cls, exploration_id, exploration_version, state_name,
-                      rule_spec_string, answer):
+                      handler_name, rule_spec_string, session_id, 
+                      time_spent_in_secs, params, answer):
         """Records an event when an answer triggers a rule."""
         # TODO(sll): Escape these args?
+        # TODO(msl): remove old answer recording models.
+
+	# In the old framework, answers are converted to unicode even when they
+	# are noot strings. This will be removed when migrating to the new
+	# framework.
         stats_models.process_submitted_answer(
```

Comment:
So per some of our discussions a couple months back, I think we established this call can be removed now. We will be recording all answers under the new storage model, so I don't think process_submitted_answer is necessary anymore.